src/main/java/microservices/userServiceBasic/repository/UserRepository.java

```diff
@@ -0,0 +1,6 @@
+package microservices.userServiceBasic.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository {
```

Comment:
Let's not use JPA for now. You can add a Collection (List, Set or Map) and implement the method save that will add the item to the Collection.